MSAL/src/native_auth/public/state_machine/error/PasswordRequiredError.swift

```diff
@@ -39,7 +39,7 @@ public class PasswordRequiredError: MSALNativeAuthError {
         super.init(message: message)
     }
 
-    init(signInPasswordError: SignInPasswordStartError) {
+    init(signInPasswordError: SignInStartError) {
```

Comment:
nit: the variable `signInPasswordError` can be renamed to match its new type

Reply:
Thanks Diego, updated ;)